OpenShift/helm/nexus-repository-manager/templates/additional-service.yaml

```diff
@@ -0,0 +1,40 @@
+{{- if .Values.service.enabled -}}
+apiVersion: v1
+kind: Service
+metadata:
+{{- if .Values.service.name }}
+  name: {{ .Values.service.name }}
+{{- else }}
+  name: {{ template "nexus.name" . }}-service
+{{- end }}
+  labels:
+{{ include "nexus.labels" . | indent 4 }}
+{{- if .Values.service.labels }}
+{{ toYaml .Values.service.labels | indent 4 }}
+{{- end }}
+{{- if .Values.service.annotations }}
+  annotations:
+{{ toYaml .Values.service.annotations | indent 4 }}
+{{- end }}
+spec:
+  ports:
+  {{- if .Values.service.portName }}
+  - name: {{ .Values.service.portName }}
+    port: {{ .Values.service.port }}
+    targetPort: {{ .Values.service.targetPort }}
+  {{- end }}
+  {{- with .Values.service.ports  }}
+{{ toYaml . | indent 2 }}
+  {{- end }}
+  selector:
+    app: {{ template "nexus.name" . }}
+    release: {{ .Release.Name }}
+  type: {{ .Values.service.serviceType }}
```

Comment:
all of the values in this file don't seem to be documented. Should be added to the readme. And should some of these be scoped under `nexus` for consistency?

Reply:
this whole file is an aberration, there is no such thing as an 'additional services' spec.

Reply:
these are as they were previously, just moved to a different template, so I don't feel *too* motivated to start adding documentation for them now.